src/services/ratingService.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable no-plusplus */
+
+import { updateOrCreate } from "./userService";
+import { Rating, Accomodation, Trips} from '../database/models';
+
+const getRatingArray = (array)=>{    // function to filter the array of rating query to get only ratings
+    const newArray = array.map(a=>a.service_rating)
+    return newArray;
+  }
+
+
+const classifyRating =(ratings)=>{
+    const serviceRating = getRatingArray(ratings)
+    const count = {};
+for (let i=0; i < serviceRating.length; i++) {
+  if (count[serviceRating[i]]) {
+    count[serviceRating[i]] += 1;
+  } else {
+    count[serviceRating[i]] = 1;
+  }
+}
+return count;
+}
+
+const round = (num) =>{
+    const m = Number((Math.abs(num) * 10).toPrecision(15));
+    return Math.round(m) / 10 * Math.sign(num);
+}
+const average = (a) => {
+  const ratings = getRatingArray(a)
+  const sum = ratings.reduce((b, c) => b + c, 0)
+  return round(sum / ratings.length);
+};
+
+
+const createRating = async (userId, accomodationId, serviceRating) => {
+  const { created } = await updateOrCreate(
+    Rating,
+    { user_id: userId, accomodation_id: accomodationId },
+    {
+      user_id: userId,
+      accomodation_id: accomodationId,
+      service_rating: serviceRating
+    })
+    return created;
+};
+const isAccomodated = async (userId, accomodationId)=>{
```

Comment:
the value or function named starting with "is" ie. is isAccomodated expect to return boolean value unless use other name